### PR TITLE
added packages/consent/consent.1.1.0

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by gasche** [comment]
It looks like the [Travis CI](https://travis-ci.org/ocaml/opam-repository/builds/312738041?utm_source=github_status&utm_medium=notification) reports many build failures. Have you had a look?

Also, I'm not sure about the name of your package. It seems to me that "consent" is a very broad English name that can be used in many, many contexts -- and it also happens to be the case that, given the current news in the English-speaking work, the name mostly evokes "sexual consent", which is maybe not the most lucky mind association you can have to your library.
Wouldn't something like "lbvs-consensus" be more descriptive?

File: packages/consent/consent.1.1.1/descr [hunk 1]
```diff
@@ -0,0 +1 @@
+Chemoinformatics software for Ligand-Based Virtual Screening (LBVS) using consensus queries (small data; no machine learning).
```
Comment (Chris00) [inline packages/consent/consent.1.1.1/descr]: IMHO: You may want to have a shorter summary line and (after a blank line) have a longer description with links for people—like me—who would need more information to understand what the package does.  (This is obviously no blocker for merging, just a small comment.)

**Comment by UnixJunkie** [comment]
I updated the descr file.

File: packages/consent/consent.1.1.1/descr (hunk 1 above)
Comment (UnixJunkie) [reply packages/consent/consent.1.1.1/descr]: done

**Comment by UnixJunkie** [comment]
There are many dependencies, so the install is fragile (the linux distro needs rdkit and open-babel, for example).
Some of the build failures:
- the parmap version in opam doesn't compile with 4.06.0 @rdicosmo 
- the camlzip I require doesn't install because of the ocaml version
- cpm doesn't install because of the ocaml version

About the name, the paper is already published and is about "consensus queries".
The software name (consent) is already in the paper and is the name of the github repository.
I don't want to rename the software because it would confuse potential users.

**Comment by gasche** [comment]
I think the remaining failures are acceptable. For 4.06, have you checked that you build correctly with `-safe-string` under an older version?

**Comment by UnixJunkie** [comment]
I confirm the package can correctly built on 4.06.0. However, since parmap and bitv currently don't have 4.06.0-ready package versions in opam, I have to opam pin them to their git repositories, which are 4.06.0 ready.

**Comment by hcarty** [comment]
I second the request for a name change and @gasche's concerns about the name, at least for the opam package. I don't think the name reflects what a user would expect. Could a prefix be added to indicate that this is a chemistry related project?

**Comment by UnixJunkie** [comment]
I can rename the opam package to lbvs-consent.
This will also be the prefix of all installed executables.
I need to create a new tag for that.

**Comment by UnixJunkie** [comment]
superseded by https://github.com/ocaml/opam-repository/pull/11014

File: packages/consent/consent.1.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
```
Comment (avsm) [inline packages/consent/consent.1.1.1/opam]: This `opam-version` should be 1.2

Comment (jpdeplaix) [reply packages/consent/consent.1.1.1/opam]: This has already been fixed #11014

**Pull request closed without merging**